static/events-api-reference.yaml

```diff
@@ -796,7 +794,7 @@ paths:
       summary: Retrieves all Subscriber Defined Filters for a given registration
       operationId: getAllSubscriberFilters
       produces:
-        - application/hal+json
+        - application/json
```

Comment:
Referring to https://github.com/AdobeDocs/adobe-io-events/pull/184/commits/ee447f3465a71c2297b27e4e618f3fe8e5af91bf#r2276554726, my point was where is the json object definition in this case or whatever response we get from CSM. That is not defined in the API spec.